### ISSUE TITLE
PMP: propose sorting of table columns and showing of done requests

### ISSUE THREAD

**Issue by ameyer-cern** [cms-PdmV/pmp#92]
Hi Justinas, all,

About the table of requests in the history view of PMP, I would like to propose the following features:
* show the list of done requests by default 
* make the table sortable by column

Thanks for your consideration
Andreas

**Comment by justinasr** [comment]
Hello,

I just merged sorting feature (#93), it is now available in pMp. You might need to clear your browser cache to get new html and javascript files.

About done requests list: initially in new pMp version it was shown all the time. However, soon we realized that it works when there are few hundred requests, but  in cases like RunIIFall17NanoAODv6 which now has ~600 submitted and ~10k requests, it takes a considerably longer time to load the page if "done list" is visible. Not everyone is interested in that list, so they are forced to wait for a thing that they not need. That's why we (for some reason I think it was Giovanni's idea) to have that list initially hidden and show only when user explicitly wants to see it.

Justinas R.
